single_offer/src/testutils.rs

```diff
@@ -57,7 +57,7 @@ impl SingleOffer {
         nonce(&mut self.env, &self.contract_id)
     }
 
-    pub fn trade(&mut self, to: &Identifier, min: u32) {
+    pub fn trade(&mut self, to: &Identifier, min: BigInt) {
```

Comment:
Nitpick: for consistency with the rest of the interface, should be `&BigInt`

Reply:
Done